third_party/helm/helm.go

```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helm
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/vladimirvivien/gexe"
+	"github.com/vladimirvivien/gexe/exec"
+	log "k8s.io/klog/v2"
+)
+
+type Opts struct {
```

Comment:
Having tons of options can be useful, but I feel that this package is becoming a wrapper for the `helm` command. In that case, would it be better to just allow test writers to just pass in command in plain text to be easier ?

Having said that, my suggestion would be to select a very restricted set of functionalities that test users can do.  For instance, instead of allowing test writers to specify a mode in plain text, maybe this package should just support just 2 or 3 modes.

Reply:
> would it be better to just allow test writers to just pass in command in plain text to be easier

I thought about this. But if the arguments are getting programatically generated during the test, the builder pattern nature that exists now will make it more cleaner and handy to implement the command.  

That said, I do agree the options can get overwhelming. The current options provided as very minimal and used in almost all helm commands to get things done. However, I am fine removing the `WithMode` in favor of a few selective helm commands. 

1. Repo Add
2. Install Charts
3. Render Charts
4. Delete Charts
5. Run Tests

These are the most primitive one I can think of. Anything else I should consider or should I even remove some from the above list ?